src/codeflare_sdk/templates/base-template.yaml

```diff
@@ -221,27 +204,6 @@ spec:
                           operator: In
                           values:
                           - "aw-kuberay"
-                initContainers:
-                # the env var $RAY_IP is set by the operator if missing, with the value of the head service name
-                - name: init-myservice
-                  image: quay.io/project-codeflare/busybox:latest
-                  command: ['sh', '-c', "until nslookup $RAY_IP.$(cat /var/run/secrets/kubernetes.io/serviceaccount/namespace).svc.cluster.local; do echo waiting for myservice; sleep 2; done"]
-                - name: create-cert
-                  image: rayproject/ray:2.7.0
-                  command:
-                  - sh
-                  - -c
-                  - cd /home/ray/workspace/tls && openssl req -nodes -newkey rsa:2048 -keyout server.key -out server.csr -subj '/CN=ray-head' && printf "authorityKeyIdentifier=keyid,issuer\nbasicConstraints=CA:FALSE\nsubjectAltName = @alt_names\n[alt_names]\nDNS.1 = 127.0.0.1\nDNS.2 = localhost\nDNS.3 = ${FQ_RAY_IP}\nDNS.4 = $(awk 'END{print $1}' /etc/hosts)">./domain.ext && cp /home/ray/workspace/ca/* . && openssl x509 -req -CA ca.crt -CAkey ca.key -in server.csr -out server.crt -days 365 -CAcreateserial -extfile domain.ext
-                  # securityContext:
-                  #   runAsUser: 1000
-                  #   runAsGroup: 1000
-                  volumeMounts:
-                  - name: ca-vol
-                    mountPath: "/home/ray/workspace/ca"
-                    readOnly: true
-                  - name: server-cert
-                    mountPath: "/home/ray/workspace/tls"
-                    readOnly: false
```

Comment:
@Fiona-Waters This initContainer should be kept because it's responsible for enabling the SSL communication between the workers and the RayClient on the Head node.